xml/src/main/java/org/ehcache/xml/service/DefaultCacheLoaderWriterConfigurationParser.java

```diff
@@ -33,6 +34,11 @@ public DefaultCacheLoaderWriterConfigurationParser() {
       CacheTemplate::loaderWriter,
       (config, loader) -> new DefaultCacheLoaderWriterConfiguration((Class<? extends CacheLoaderWriter<?, ?>>) getClassForName(config, loader)),
       CacheType::getLoaderWriter, CacheType::setLoaderWriter,
-      config -> new CacheLoaderWriterType().withClazz(config.getClazz().getName()));
+      config -> {
+        if (config.getInstance() != null) {
```

Comment:
And I agree with @albinsuresh that you could have a `checkNoConcreteInstance` util method somewhere